File: test/cs/case31_math_extended.cs
using System.Collections.Generic;
using System.IO;
using System;

public static class Program
{
    public static void main()
    {
        Pytra.CsModule.py_runtime.print((Math.Abs(Math.Tan(0.0)) < 1e-12));
        Pytra.CsModule.py_runtime.print((Math.Abs((Math.Log(Math.Exp(1.0)) - 1.0)) < 1e-12));
        Pytra.CsModule.py_runtime.print(Pytra.CsModule.py_runtime.py_int(Math.Log10(1000.0)));
        Pytra.CsModule.py_runtime.print(Pytra.CsModule.py_runtime.py_int((Math.Abs((-3.5)) * 10.0)));
        Pytra.CsModule.py_runtime.print(Pytra.CsModule.py_runtime.py_int(Math.Ceiling(2.1)));
        Pytra.CsModule.py_runtime.print(Pytra.CsModule.py_runtime.py_int(Math.Pow(2.0, 5.0)));
    }

    public static void Main(string[] args)
    {
        main();
    }
}

File: test/js/case31_math_extended.js
// このファイルは自動生成です（Python -> JavaScript native mode）。

const __pytra_root = process.cwd();
const py_runtime = require(__pytra_root + '/src/js_module/py_runtime.js');
const py_math = require(__pytra_root + '/src/js_module/math.js');
const py_time = require(__pytra_root + '/src/js_module/time.js');
const pathlib = require(__pytra_root + '/src/js_module/pathlib.js');
const { pyPrint, pyLen, pyBool, pyRange, pyFloorDiv, pyMod, pyIn, pySlice, pyOrd, pyChr, pyBytearray, pyBytes, pyIsDigit, pyIsAlpha } = py_runtime;
const { perfCounter } = py_time;
const math = require(__pytra_root + '/src/js_module/math.js');

function main() {
    pyPrint(((math.fabs(math.tan(0.0))) < (1e-12)));
    pyPrint(((math.fabs(((math.log(math.exp(1.0))) - (1.0)))) < (1e-12)));
    pyPrint(Math.trunc(Number(math.log10(1000.0))));
    pyPrint(Math.trunc(Number(((math.fabs((-(3.5)))) * (10.0)))));
    pyPrint(Math.trunc(Number(math.ceil(2.1))));
    pyPrint(Math.trunc(Number(math.pow(2.0, 5.0))));
}
main();
